src/theme/base.css

```
@@ -28,6 +28,12 @@
     --color-accent: var(--input-bg);
     --color-accent-foreground: var(--text-primary);
 
+    /* May 2025 Koto design colors */
+    --color-v-charcoal: var(--v-charcoal);
```

Follow the suggestion above, and rename the var to the color design name
```suggestion
    --color-bg-80: var(--bg-80);
```

---

I was going to do it in the Colors PR since this one is already very huge